base.py

```diff
@@ -151,6 +162,9 @@ def run(self):
             
         except (NoSuchWindowException, WebDriverException):
             print('\nThe process was stopped, because the browser was closed.')
```

Comment:
As I remember we talked about rephrasing this text because it is no browser window in a docker container.

Reply:
It's the browser window being rendered in X virtual framebuffer.

Reply:
Before docker, the exception meant that the user closed the Browser window. So I suggest making the text more specific for example, `\nThe process was stopped because the web driver exception has occurred.`. Also because even not in docker it often not connected with user's activity.